src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -52,6 +64,53 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
           role: "rowheader",
           "aria-label": "Type",
         },
+        {
+          content: (
+            <div>
+              {device.type == "disk" ? (
+                isRootDisk(device as FormDevice) ? (
+                  <>
+                    <strong>Pool: </strong>
+                    <>{device.pool}</>
+                  </>
+                ) : (
+                  <>
+                    <strong>Storage Volume:</strong>
```

Comment:
for non-root disk devices, show two chips displaying storage pool and volume. Use `ResourceLink` for this